sybl/client/job_config.py

```
@@ -11,6 +11,9 @@ def __init__(
         self.max_timeout = timeout
 
     def compare(self, job_config: Dict) -> bool:
+        if "timeout" not in job_config and "prediction_type" not in job_config:
+            return False
```

Should this show an error of some kind instead of failing silently?

---

I couldn't work out how to cleanly import the logging so ignored it